Test Runner/UI Tests/FeatureFormViewTests.swift

```diff
@@ -1398,6 +1398,249 @@ final class FeatureFormViewTests: XCTestCase {
 #endif
         XCTAssertEqual(barcodeValidationString.label, "Maximum 50 characters")
     }
+    
+    func testCase_12_1() {
+        let app = XCUIApplication()
+        let assetGroup = app.staticTexts["Asset group"]
+        let elementTitle = app.staticTexts["Associations"]
+        let fieldValue = app.staticTexts["Asset group Read Only Input"]
+        let formTitle = app.staticTexts["Electric Distribution Device"]
+        let filterResults1 = app.staticTexts["Connected"]
+        let filterResults2 = app.staticTexts["Structure"]
+        let filterResults3 = app.staticTexts["Container"]
+        let networkSourceGroup1 = app.staticTexts["Electric Distribution Junction"]
+        let networkSourceGroup2Button = app.buttons["Electric Distribution Device, 2"]
+        let utilityElement1Button = app.buttons["Fuse - 3907, Fuse, Terminal: Single Terminal"]
+        let utilityElement2Button = app.buttons["Fuse - 1392, Fuse, Terminal: Single Terminal"]
+
+        openTestCase()
+        assertFormOpened(titleElement: formTitle)
+
+        XCTAssertTrue(
+            elementTitle.exists,
+            "The element \"Associations\" doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            filterResults1.exists,
+            "The filter result \"Connected\" doesn't exist."
+        )
+
+        XCTAssertTrue(
+            filterResults2.exists,
+            "The filter result \"Structure\" doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            filterResults3.exists,
+            "The filter result \"Container\" doesn't exist."
+        )
+        
+        filterResults1.tap()
+
+        XCTAssertTrue(
+            networkSourceGroup1.exists,
+            "The network source group \"Electric Distribution Junction\" doesn't exist."
+        )
+        
+        XCTAssertTrue(
+            networkSourceGroup2Button.waitForExistence(timeout: 5),
+            "The network source group \"Electric Distribution Device\" doesn't exist."
+        )
+
+        networkSourceGroup2Button.tap()
+
+        XCTAssertTrue(
+            utilityElement1Button.waitForExistence(timeout: 30),
+            "Feature \"Fuse - 3097\" failed to appear after 30 seconds."
+        )
+        
+        XCTAssertTrue(
+            utilityElement2Button.exists,
+            "The utility element \"Fuse - 1392\" doesn't exist."
+        )
+
+        utilityElement1Button.tap()
+        
+        // Open new form
+        assertFormOpened(titleElement: formTitle)
+        
+        XCTAssertTrue(
+            assetGroup.exists,
+            "The asset group \"Asset group\" doesn't exist."
+        )
+
+        XCTAssertEqual(
+            fieldValue.label,
+            "Fuse"
+        )
+    }
+    
+    // Test case 12.2: Associations show fraction along edge - NEEDS DATA
+//    func testCase_12_2() {
+//    }
```

Comment:
Given the latest info in Apollo 1153, I'm thinking we should just remove this and deprecate the test case.